bool isMatch		#True if match found
float32 matchProb	#Matching probability
int32 kfId1		#Keyframe id for camera 1
int32 kfId2		#Keyframe id for camera 2
